### DISCUSSION TITLE
Opens on IpadOS but not on IOS?

### DISCUSSION THREAD

**Discussion by Schurla74** [mvdkwast/obsidian-copy-as-html#49] [Q&A]
Hi all, I know, this fantastic plug-in does not work on mobile devices, but my Obsidian app opens up on the iPad when the Plug-In is activated, but it fails to open on the iPhone. 
Am I the only one experiencing this, cause that would be a pity, as I so much love the plug-in on my Mac.
Thanks for your help, cheers
Walter

**Comment by mvdkwast** [comment]
Hi Walter, sorry for the late reply. Mobile support has become a common request, I created issue #61 as a feature request. No guarantees though (I'm not sure why it didn't work in the first place).

**Comment by mvdkwast** [comment]
Hi @Schurla74, if you're a developer, or at least confident enough to manually build and update the plugin on iOS, you can checkout the `feature/mobile` branch. I have no way to test on iOS, and clipboard won't work on Android, so I'm pretty much stuck here. Also see #61 .
